ui/ui-components/components/export/List.tsx

```
@@ -52,14 +55,55 @@ export default function ExportsList(props: Props) {
     }
   }
 
+  const retryExport = useCallback(
+    async (_export: Models.ExportType, index: number) => {
+      setLoading(true);
+      try {
+        const { count } = await client.requestAction<ExportsRetryFailed>(
+          "get",
+          `/exports/retryFailed`,
+          {
+            destinationId: _export.destination.id,
+            startTimestamp: _export.createdAt,
+            endTimestamp: _export.createdAt,
+          },
+          { useCache: false }
+        );
+        if (count) {
+          successHandler.publish({ message: "Export Retried" });
```

Not sure why, but we use `(successHandler | errorHandler).set` in core. What's the difference between set and publish?

---

```
  set(data: T) {
    this.publish(data);
  }
```
Apparently nothing!

---

I'm not sure my `catch` code path will never be traversed since we eat all errors client-side.